mne/viz/utils.py

```diff
@@ -2328,30 +2330,63 @@ def _plot_masked_image(
 
 
 @fill_doc
-def _make_combine_callable(combine):
+def _make_combine_callable(
+    combine,
+    *,
+    axis=1,
+    valid=("mean", "median", "std", "gfp"),
+    ch_type=None,
+    keepdims=False,
```

Comment:
addition of `axis`, `valid`, and `keepdims` are to make this function useful for TFRs (#11282)